tests/test_app.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import mock
+import requests
+import app
+from unittest import TestCase
+
+
+class TestApp(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        app.app.testing = True
+        cls.client = app.app.test_client()
+
+
+class TestGetHospitalCapacity(TestApp):
+    route = '/local-hospital-capacity'
+
+    @mock.patch('app.requests.get')
+    def test_get_hospital_capacity__raises_http_error(self, mock_get):
```

Comment:
Is the double _ a convention for tests? Just curious, I don't think I've ever seen it before.